keyboards/massdrop/ctrl/keymaps/r-pufky/layout-layer0.json

```diff
@@ -0,0 +1,6 @@
+[{a:5},"Esc",{x:1},"F1","F2","F3","F4",{x:0.5},"F5","F6","F7","F8",{x:0.5},"F9","F10","F11","F12",{x:0.25},"PrtSc","ScrLk","Pause"],
```

Comment:
These are KLE renders, right?

Reply:
They should probably be deleted.

Reply:
Correct, they are the json config for http://www.keyboard-layout-editor.com/ to reproduce layout render.